Example.hs

```diff
@@ -19,6 +21,10 @@ A minimal environment (to support the code generated by desugaring) is provided
 import DA.Internal.Desugar
 import GHC.Types
 
+interface TheInterface where
+  myInternalValue : Int
+  viewtype Int
```

Comment:
we could just call this `view` I think. Don’t care that much about the syntax so happy to leave that to your team.

Reply:
makes sense, then let’s stick to this.

Reply:
I think this is okay for now